confgenerator/confmerger.go

```diff
@@ -20,7 +20,7 @@ func MergeConfFiles(userConfPath, platform string, builtInConfStructs map[string
 	builtInStruct := builtInConfStructs[platform]
 
 	// Read the built-in config file.
-	original, err := builtInStruct.DeepCopy(platform)
+	template, err := builtInStruct.DeepCopy(platform)
```

Comment:
[Optional] I'd call this `merged` or `result`, FWIW. Eventually, we should probably turn `mergeConfigs` into a functional update (rather than in-place) and do the `DeepCopy` there.

Reply:
Done. Naming is hard :(